<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWin</class>
 <widget class="QMainWindow" name="LoginWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
	border-image: url(:/蓝.jpg);
}

QLineEdit#userlineEdit,#pwdlineEdit{
		border-radius:5px;
		border:1px solid;
		
	selection-background-color: rgb(138, 226, 52);
	selection-color: rgb(46, 52, 54);
}
QLineEdit:focus{
	
	
}

QPushButton:focus{
	background-color: rgb(252, 233, 79);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>70</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>60</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Ubuntu Mono</family>
              <italic>true</italic>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="text">
             <string>用户:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="userlineEdit">
            <property name="text">
             <string>202001</string>
            </property>
            <property name="maxLength">
             <number>64</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>密码:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pwdlineEdit">
            <property name="text">
             <string>abc123</string>
            </property>
            <property name="maxLength">
             <number>16</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="registerBtn">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px solid#428aff;</string>
          </property>
          <property name="text">
           <string>注册</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginBtn">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px solid#428aff;</string>
          </property>
          <property name="text">
           <string>登陆</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitBtn">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px solid#428aff;</string>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,215,0);</string>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>欢迎使用XXX系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
